pkg/couchdb/couchdb.go

```diff
@@ -601,8 +601,14 @@ func UpdateDocWithOld(db prefixer.Prefixer, doc, oldDoc Doc) error {
 		return err
 	}
 	doctype := doc.DocType()
-	if id == "" || doc.Rev() == "" || doctype == "" {
-		return fmt.Errorf("UpdateDoc doc argument should have doctype, id and rev")
+	if doctype == "" {
+		return fmt.Errorf("UpdateDoc: doctype is missing")
+	}
+	if id == "" {
+		return fmt.Errorf("UpdateDoc: id is missing")
+	}
+	if doc.Rev() == "" {
+		return fmt.Errorf("UpdateDoc: no rev should be given")
```

Comment:
Shouldn't the message actually be `rev is missing`?

Reply:
yep, good catch!